meta-rz-bsp/conf/machine/include/rz-common.inc

```diff
@@ -1,7 +1,7 @@
 require conf/machine/include/soc-family.inc
 
 MACHINE_FEATURES = ""
-IMAGE_FSTYPES:append = " tar.gz"
+IMAGE_FSTYPES:append = " tar.gz wic.gz wic.bmap"
```

Comment:
We should move this change to `meta-rz-bsp/conf/machine/include/rzg2l-family.inc` until we add support for the `hihope-rzg2h` machine, otherwise we'll get a build error complaining that `WKS_FILE` isn't set.

Reply:
You are right, will do.